pq-crypto/kyber_r2_90s/Makefile

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+SRCS=aes256ctr.c cbd.c indcpa.c ntt.c poly.c polyvec.c reduce.c verify.c aes_c.c sha2_c.c kyber_r2_kem.c
```

Comment:
Tracked down the build failures to a typo on this line. 

Replace `kyber_r2_kem.c` with `kyber_r2_90s_kem.c` on this line.